### ISSUE TITLE
Add action to automatically add issues and PRs to a project

### ISSUE THREAD

**Issue by cjyetman** [RMI-PACTA/demo_actions#26]
I might be the only one that uses it, but I have a [board](https://github.com/orgs/RMI-PACTA/projects/8/views/9) that is meant to track all issues and PRs related to Transition Monitor across many repos. It's only really feasible because I setup GH actions in each relevant repo to automatically add any newly created issues or PRs, like this https://github.com/RMI-PACTA/pacta.portfolio.allocate/blob/main/.github/workflows/add-prs-and-issues-to-project.yml

**Comment by jdhoffa** [comment]
I have a similar thing going for `r2dii` here:
https://github.com/orgs/RMI-PACTA/projects/10

I think we should align on EITHER doing this, or #23 (otherwise we are really risking falling out of sync/ maintaining two of the same thing)

**Comment by jdhoffa** [comment]
Yeah fair enough. Just want to make sure you don't spend all of the time that you want to put towards Project Management on beautifying that GitHub board, and have that result in their ADO counterparts being left to rot (playing Matze's advocate here a bit seeing as he's not on GH lol) 

but if you love PM so much you want to have two boards then that works 😂

**Comment by cjyetman** [comment]
I don't love PM so much, but I appreciate the ability to see related issues and PRs across multiple repos that are relevant, and be able to interact with them directly. If all the relevant code was in one repo and not spread out across numerous separate repos, I'd probably be fine with using just that repo's Issues and PRs tabs.

**Comment by jdhoffa** [comment]
Fair enough!

**Comment by jdhoffa** [comment]
Tagging as `agile` though granted it's not exactly/ specifically agile, but agile-adjacent.
